### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by happyleavesaoc** [happyleavesaoc/python-snapcast#12]
We now use asyncio. Update the README as necessary, and add some more examples of usage.

**Comment by ryanrdetzel** [comment]
Yeah, I'm stuck on this. :-) A little help on how to use this would be great.

**Comment by happyleavesaoc** [comment]
Couple places you can look in the interim:

1) Tests
2) Home Assistant component: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/snapcast.py
